gcc/rust/rust-session-manager.h

```diff
@@ -211,6 +218,26 @@ struct CompileOptions
     crate_name = std::move (name);
     return true;
   }
+
+  bool set_edition (std::string edition_value)
+  {
+    auto ret_val = true;
+    if (edition_value == "2015")
+      edition = E2015;
```

Comment:
Should the 2015 edition be rejected until name resolution according to the 2015 edition rules is implemented?

Reply:
That's a good point. Implementing proper handling of `pub(restricted)` also falls in that same category of behavior that we may want to gate for now. @philberty what do you think?

Reply:
Yeah that sounds good to me.